Examples/Catalog/macOS/Source/ViewControllers/ModelCollectionExampleCollectionViewController.swift

```diff
@@ -0,0 +1,38 @@
+import Cocoa
+import CatalogCore
+import Pilot
+import PilotUI
+
+public final class ModelCollectionExampleCollectionViewController: CollectionViewController {
+    
+    // MARK: Public
+    
+    public static func make(with context: CatalogContext) -> ModelCollectionExampleCollectionViewController {
+        let layout = CollectionViewListLayout()
+        return ModelCollectionExampleCollectionViewController(
+            model: CommonModelCollections.makeModelCollectionExamples(),
+            modelBinder: DefaultViewModelBindingProvider(),
+            viewBinder: ModelCollectionExampleViewBinder(),
+            layout: layout,
+            context: context)
+        
+    }
+    
+    // MARK: NSViewController
+    
+    public override func viewDidLoad() {
+        super.viewDidLoad()
+        backgroundColor = .white
+    }
+}
+
+// TODO:(wkiefer) Replace when Pilot has a block-based view binding provider.
```

Comment:
We do have `StaticViewBindingProvider(type: ExampleView.self)` which would work here fwiw

Reply:
done.